packages/governance/src/committeeRegistrar.js

```diff
@@ -0,0 +1,84 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { makeNotifierKit } from '@agoric/notifier';
+import { E } from '@agoric/eventual-send';
+import { makeStore } from '@agoric/store';
+import { natSafeMath } from '@agoric/zoe/src/contractSupport';
+
+const { add, multiply } = natSafeMath;
+
+const start = zcf => {
+  const allQuestions = makeStore('Question');
+
+  const makeCommitteeVoterInvitation = index => {
+    const handler = voterSeat => {
+      return Far(`voter${index}`, {
+        castBallot: ballot =>
+          E(allQuestions.get(ballot.question)).submitVote(voterSeat, ballot),
+      });
+    };
+
+    return zcf.makeInvitation(handler, `Voter${index}`);
+  };
+
+  const createRegistrar = (name, count) => {
+    const { notifier, updater } = makeNotifierKit();
+
+    const invitations = [];
+    for (let i = 0; i < count; i += 1) {
+      invitations[i] = makeCommitteeVoterInvitation(i);
+    }
+
+    // This could be parameterized. For now, we check that at least half voted.
+    const quorumChecker = Far('checker', {
```

Comment:
Why not have this be a parameter of the ballot counter, rather than an object whose methods must be called asynchronously?